README.md

```diff
@@ -4,6 +4,17 @@
 Database uptime can be checked at this address:  
 [https://kuma.arturkamieniecki.pl/status/pieskiuw](https://kuma.arturkamieniecki.pl/status/pieskiuw)
 
+## Webscraper application
+The webscraper application uses threads not to halt the main django process. Because it uses them to update the database `celery` and `redis` is needed.
+The redis server needs to be set up. Please follow the redis documentation to install the server. If run on ubuntu the redis server can be installed using:  
+```bash
+$ apt install redis 
+```
+The celery process also needs to be started with the Django server:
+```bash
+$ celery -A pieskiUW worker --loglevel=info --concurrency 1 -E
```

Comment:
I think that this documentation could be improved but it isn't required because docker configuration is planned